config.go

```diff
@@ -95,26 +97,44 @@ func GetOSConfigPath(homePath string) string {
 	return filepath.Join(configDir, "eget", "eget.toml")
 }
 
-func InitializeConfig() *Config {
+func InitializeConfig() (*Config, error) {
 	var err error
 	var config Config
 
 	homePath, _ := os.UserHomeDir()
 	appName := "eget"
 
 	if configFilePath, ok := os.LookupEnv("EGET_CONFIG"); ok {
-		config, err = LoadConfigurationFile(configFilePath)
+		if config, err = LoadConfigurationFile(configFilePath); err != nil {
+			if !errors.Is(err, os.ErrNotExist) {
+				return &config, fmt.Errorf("error loading config: %v: %w", configFilePath, err)
+			}
```

Comment:
I think the two if statements can be merged, and we can just return `nil, fmt.Errorf("config: %w", err)` for simplicity.

```
if config, err = LoadConfigurationFile(configFilePath); err != nil && !errors.Is(err, os.ErrNotExist) {
    return nil, fmt.Errorf("config: %w", err)
}
```

Reply:
Sounds good to me.
I separated them mostly to not trigger any "line too long" linter.

Are you sure is a good idea to remove the actual filename from the error message?
I think is important to also report which config file the error is about, so the user knows where to look for.

Reply:
Sure, including the path is fine. In that case I think there is enough info to just put `fmt.Errorf("%s: %w", configFilePath, err)`.

Reply:
Done!
Errors look like this now:
```
$ eget test/test
/home/hhromic/.config/eget/eget.toml: toml: line 4 (last key "global.file"): incompatible types: TOML value has type []interface {}; destination has type string
```